embedchain/config/vectordbs/LanceDBConfig.py

```diff
@@ -0,0 +1,39 @@
+import os
+from typing import Dict, List, Optional, Union
+
+from embedchain.config.vectordbs.BaseVectorDbConfig import BaseVectorDbConfig
+from embedchain.helper.json_serializable import register_deserializable
+
+@register_deserializable
+class LanceDBConfig(BaseVectorDbConfig):
```

Comment:
Hey, support for creating apps using yaml was recently added in https://github.com/embedchain/embedchain/pull/787. Would be nice to  ensure this PR adheres to those changes?